### PR TITLE
Add options to create/remove groups via occ

### PR DESCRIPTION
**Pull request by denismosolov**
Add options to create and remove groups via command line.
Requested in #8317
Syntax:
`occ group:add groupid` creates a new group, groupid - the new groups name
`occ group:delete groupid` removes group, cannot remove the group named 'admin'

@tterranigma , @EpeR1 what do you think?

Signed-off-by: Denis Mosolov <denismosolov@gmail.com>

### CODE REVIEW

**Comment by rullzer** [comment]
Nice stuff!

File: core/Command/Group/Add.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?php
+/**
```
Comment (rullzer) [inline core/Command/Group/Add.php]: Could you make the file strict?

```php
declare(strict_types=1);
```

File: core/Command/Group/Add.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+	protected function execute(InputInterface $input, OutputInterface $output) {
+		$groupName = $input->getArgument('name');
+		$group = $this->groupManager->get($groupName);
+		if ($group) {
+			$output->writeln('<error>Group "' . $groupName . '" already exists.</error>');
+			return 1;
+		} else {
+			$this->groupManager->createGroup($groupName);
+			$group = $this->groupManager->get($groupName);
```
Comment (rullzer) [inline core/Command/Group/Add.php]: `createGroup` already returns the group no need for the get.

**Review by rullzer — DISMISSED** [review]
We require you to sign off on your commits.

```
git commit --amend -s
```

Should do the trick


Also, could you run:

```
bash build/autoloaderchecker.sh
```

To make sure the autoloader is up to date?

**Review by ChristophWurst — APPROVED** [review]
Tested and works :rocket:

**Review by rullzer — APPROVED** [review]

**Comment by denismosolov** [comment]
Thanks guys!

**Comment by ChristophWurst** [comment]
> denismosolov merged commit a0b84bc into nextcloud:master

https://github.com/nextcloud/server/pull/10334#issuecomment-415689196 -> @MorrisJobke @rullzer shall we revert this change? This is a new feature and we're past feature freeze.

**Comment by rullzer** [comment]
I'm fine with having this. Isolated new commands.

**Comment by MorrisJobke** [comment]
@denismosolov In the future please wait for the approval for merge by others if it is stated like this in the comments. We are currently shortly before the RC 1 release and want to have this stable.

**Comment by MorrisJobke** [comment]
For this one it's fine.

**Comment by denismosolov** [comment]
Sorry for the accident. I didn't know what to do after getting 2 reviews. Just clicked the merge button because UI alowed me to do that.

**Pull request merged**